sno/diff_output.py

```diff
@@ -420,6 +440,9 @@ def geojson_row(row, pk_value, change=None):
         v = row[k]
         if isinstance(v, bytes):
             g = gpkg_geom_to_ogr(v)
+            if geometry_transform is not None:
+                # reproject
+                g.Transform(geometry_transform)
             f["geometry"] = json.loads(g.ExportToJson())
```

Comment:
We should really fix the GDAL Python bindings to avoid this encode+decode dance 😄  File a feature request? 🤞

Reply:
Would it necessarily be faster? It would require the bindings to produce a ton of python objects, whereas at present they just handle one string. I'm happy with leaving this as-is until we identify it as some kind of bottleneck

Reply:
>  It would require the bindings to produce a ton of python objects

Sure, but the user ends up with those same Python objects _anyway_.

Emitting JSON is relatively trivial, but parsing _anything_ is much less so.

Reply:
okay, have filed this as OSGeo/gdal#2877